.travis.yml

```diff
@@ -1,13 +1,24 @@
+sudo: required
+dist: trusty
+group: edge
+
 language: go
 go:
     - 1.6.x
     - 1.7.x
     - 1.8.x
     - tip
+
 install:
     - go get -v golang.org/x/tools/cmd/cover
+    - sudo add-apt-repository -y ppa:opencpu/opencpu-1.6
```

Comment:
It looks like 2.0 breaks the API for starting the server. This is unfortunate. Can I clarify that the change from `opencpu$start(port)` to `ocpu_start_server(port = 5656, root = "/ocpu", workers = 2,
preload = NULL, on_startup = NULL, no_cache = FALSE)` happened in 2.0.0?